.github/workflows/publish.yml

```
@@ -0,0 +1,136 @@
+    steps:
+    - name: Download all the dists
+      uses: actions/download-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+
+    - name: Publish distribution 📦 to TestPyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        repository-url: https://test.pypi.org/legacy/
+        token: ${{ secrets.TEST_PYPI_TOKEN }}
```

actionlint (1.7.7)

input "token" is not defined in action "pypa/gh-action-pypi-publish@release/v1". available inputs are "attestations", "packages-dir", "packages_dir", "password", "print-hash", "print_hash", "repository-url", "repository_url", "skip-existing", "skip_existing", "user", "verbose", "verify-metadata", "verify_metadata"

(action)